tests/h/views/admin/users_test.py

```diff
@@ -8,7 +8,7 @@
 import pytest
 
 from h.services.annotation_stats import AnnotationStatsService
-from h.services.delete_user import DeleteUserService, UserDeleteError
+from h.services.delete_user import DeleteUserService
```

Comment:
It's nice to see this simplified a little (custom exceptions are good, but even better when we don't need 'em)!